guides/rest_doc.md

```diff
@@ -21,6 +21,10 @@
 ## App
 * [kintoneUtility.rest.getFormFields](#getFormFields)
 * [kintoneUtility.rest.getFormLayout](#getFormLayout)
+* [kintoneUtility.rest.postDeploy](#postDeploy)
+* [kintoneUtility.rest.getDeploy](#getDeploy)
+* [kintoneUtility.rest.getCustomize](#getCustomize)
```

Comment:
Because customize is a verb, so the function name should be changed
getCustomize => getCustomization
putCustomize => updateCustomization